### PR TITLE
feature/run-chromedriver-tests-in-docker

### PR DESCRIPTION
**Pull request by dabeeeenster**

### CODE REVIEW

**Comment by matthewelwell** [comment]
Not sure what to do with this one @dabeeeenster. To get it working on M1 you'd need to at least add `platform: linux/amd64` but even then, there's an issue with ChromeDriver. We can either merge it with the caveat that it doesn't work on M1 or we need to have a look at it - maybe one to discuss with @kyle-ssg when he's back.

**Comment by dabeeeenster** [comment]
I think merge for now - its still useful for non-m1

File: frontend/docker-compose-e2e-tests.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            - db:db
+
+    frontend:
+        build: 
+            context: .
+            dockerfile: Dockerfile.e2e
+        environment:
+            E2E_TEST_TOKEN_DEV: some-token
+            ENABLE_INFLUXDB_FEATURES: 'false'
+            EMAIL_BACKEND: django.core.mail.backends.smtp.EmailBackend
```
Comment (matthewelwell) [inline frontend/docker-compose-e2e-tests.yml]: This isn't needed on the FE.

File: frontend/Dockerfile.e2e [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM node:12 AS production
```
Comment (matthewelwell) [inline frontend/Dockerfile.e2e]: I think the `AS production` is unnecessary here?

File: frontend/Dockerfile.e2e [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN mkdir /srv/bt && chown node:node /srv/bt
```
Comment (matthewelwell) [inline frontend/Dockerfile.e2e]: Why `/srv/bt` ?

File: frontend/Dockerfile.e2e (hunk 2 above)
Comment (dabeeeenster) [reply frontend/Dockerfile.e2e]: Good point

File: frontend/Dockerfile.e2e (hunk 3 above)
Comment (dabeeeenster) [reply frontend/Dockerfile.e2e]: renamed

File: frontend/Dockerfile.e2e [hunk 4]
```diff
@@ -0,0 +1,20 @@
+FROM node:12
+
+RUN mkdir /srv/flagsmith && chown node:node /srv/flagsmith
+
+RUN apt-get update
+RUN wget https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_91.0.4472.114-1_amd64.deb
```
Comment (kyle-ssg) [inline frontend/Dockerfile.e2e]: https://github.com/Flagsmith/flagsmith/pull/548/files#diff-88b40629df399d3611837cb436b6d43f743791f1f5944ea002bb76bf3713046aR61

Comment (kyle-ssg) [reply frontend/Dockerfile.e2e]: After linked PR is merged (or if this one is first) I think we'd want to update this to match.

**Pull request merged**